e2e/testconfig/testconfig.go

```diff
@@ -363,7 +363,6 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
 
 	tmTomlOverrides["log_level"] = cometCfg.LogLevel // change to debug in ~/.ibc-go-e2e-config.json to increase cometbft logging.
 	configFileOverrides["config/config.toml"] = tmTomlOverrides
-
 	useNewGenesisCommand := cc.Binary == icadBinary && testvalues.IcadNewGenesisCommandsFeatureReleases.IsSupported(cc.Tag)
```

Comment:
This bool logic needs to be extended to do feature release support for the new cmd structure for simd right?

Reply:
I think we can add:
```go
if cc.Binary == simdBinary && testvalues.SimdNewGenesisCommandsFeatureReleases.IsSupported(cc.Tag) {
    useNewGenesisCommand = true
}
```

and add this to `e2e/testvalues/values.go
```go
// SimdNewGenesisCommandsFeatureReleases represents the releases the simd binary started using the new genesis command.
var SimdNewGenesisCommandsFeatureReleases = semverutil.FeatureReleases{
    MajorVersion: "v8",
}
```